tests/integration_tests/test_gbm.py

```diff
@@ -174,6 +174,13 @@ def test_local_gbm_number(tmpdir, local_backend):
     run_test_gbm_number(tmpdir, local_backend)
 
 
+@pytest.mark.distributed
+def test_ray_gbm_number_remote(tmpdir, ray_backend, ray_cluster_5cpu):
```

Comment:
Would be good to add a comment here explaining why we want to test this specifically.